.travis.yml

```diff
@@ -3,12 +3,6 @@ dist: xenial
 language: minimal
 services:
 - docker
-env:
-- OS_NAME: linux
-before_script:
-- go get golang.org/x/tools/cmd/cover
-- go get github.com/mattn/goveralls
```

Comment:
We're running these commands inside Docker. We want to use the same version of Go for all the operations that pertain to Go code. We have seen build failures otherwise, because `goveralls` seems to import Go code.

See for example https://travis-ci.org/ooni/probe-cli/builds/629707378#L1213